backend/src/test/java/com/woowacourse/levellog/application/OAuthServiceTest.java

```diff
@@ -1,77 +1,50 @@
 package com.woowacourse.levellog.application;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.BDDMockito.given;
 
-import com.woowacourse.levellog.authentication.application.OAuthService;
-import com.woowacourse.levellog.authentication.domain.OAuthClient;
-import com.woowacourse.levellog.authentication.dto.GithubCodeRequest;
+import com.woowacourse.levellog.authentication.dto.GithubCodeDto;
 import com.woowacourse.levellog.authentication.dto.GithubProfileDto;
-import com.woowacourse.levellog.authentication.dto.LoginResponse;
-import com.woowacourse.levellog.authentication.support.JwtTokenProvider;
-import com.woowacourse.levellog.member.application.MemberService;
+import com.woowacourse.levellog.authentication.dto.LoginDto;
 import com.woowacourse.levellog.member.dto.MemberCreateDto;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.transaction.annotation.Transactional;
 
-@SpringBootTest
-@Transactional
 @DisplayName("OAuthService의")
-class OAuthServiceTest {
-
-    @MockBean
-    private OAuthClient oAuthClient;
-
-    @Autowired
-    private MemberService memberService;
-
-    @Autowired
-    private JwtTokenProvider jwtTokenProvider;
-
-    @Autowired
-    private OAuthService oAuthService;
+class OAuthServiceTest extends ServiceTest {
 
     @Nested
     @DisplayName("login 메서드는")
     class login {
 
         @Test
         @DisplayName("첫 로그인 시 회원가입하고 id, 토큰, 이미지 URL를 반환한다.")
-        void loginFirst() {
-            // given
-            given(oAuthClient.getAccessToken("githubCode")).willReturn("accessToken");
-            given(oAuthClient.getProfile("accessToken")).willReturn(
-                    new GithubProfileDto("12345", "로마", "imageUrl"));
-
+        void login_first_signUp() throws Exception {
             // when
-            final LoginResponse tokenResponse = oAuthService.login(new GithubCodeRequest("githubCode"));
+            final GithubProfileDto request = new GithubProfileDto("1234", "릭", "rick.org");
+            final LoginDto tokenResponse = oAuthService.login(
+                    new GithubCodeDto(objectMapper.writeValueAsString(request)));
             final String payload = jwtTokenProvider.getPayload(tokenResponse.getAccessToken());
             final Long savedMemberId = memberService.findMemberById(Long.parseLong(payload))
                     .getId();
 
             // then
             assertThat(Long.parseLong(payload)).isEqualTo(savedMemberId);
-            assertThat(tokenResponse.getProfileUrl()).isEqualTo("imageUrl");
+            assertThat(tokenResponse.getProfileUrl()).isEqualTo("rick.org");
             assertThat(tokenResponse.getId()).isNotNull();
```

Comment:
```suggestion
            assertAll(
                () -> assertThat(Long.parseLong(payload)).isEqualTo(savedMemberId);
                () -> assertThat(tokenResponse.getProfileUrl()).isEqualTo("rick.org");
                () -> assertThat(tokenResponse.getId()).isNotNull();
            );  
```
하나의 테스트에서 여러개의 assert문을 사용하는 경우 assertAll()을 사용해보는건 어떨까요?

참고) [StackOverflow - assertAll vs multiple assertions in JUnit5](https://stackoverflow.com/questions/40796756/assertall-vs-multiple-assertions-in-junit5)